RobotCode/src/main/java/frc/robot/subsystems/ShooterMountSubsystem.java

```diff
@@ -0,0 +1,79 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.lib.core.motors.TeamSparkMAX;
+import frc.robot.constants.Ports;
+
+public class ShooterMountSubsystem extends SubsystemBase {
+
+	public TeamSparkMAX mainMotor;
+	public TeamSparkMAX followMotor;
+
+
+	public double currentRotation = 0.0;
+
+	private double originalRotation = 0.0;
+	private double goalRotation = 0.0;
+
+   private static ShooterMountSubsystem instance;
```

Comment:
We don't need an instance variable if we aren't accessing the instance from anywhere outside of that instance.